controls/pcidss_ocp4.yml

```diff
@@ -1,6 +1,6 @@
 policy: PCI-DSS
 title: Configuration Recommendations of a GNU/Linux System
-id: pcidss
+id: pcidss_ocp4
```

Comment:
Does renaming the ID from `pcidss` have any impact on the references in the rules that still use `pcidss` ?

Reply:
This is the ID of the controls and not the benchmark. It doesn't affect references